components/accordion/src/_mixin.scss

```diff
@@ -13,7 +13,21 @@
 }
 
 @mixin utrecht-accordion__section {
+  border-block-start-color: var(--utrecht-accordion-section-border-color);
+  border-block-start-style: solid;
+  border-block-start-width: var(--utrecht-accordion-section-border-width);
   break-inside: avoid;
+  &:is(:last-child, :only-child) {
+    border-block-end-color: var(--utrecht-accordion-section-border-color);
+    border-block-end-style: solid;
+    border-block-end-width: var(--utrecht-accordion-section-border-width);
+  }
+  &:not(:last-child) {
+    border-block-end-color: transparent;
+  }
+  &:hover {
+    --utrecht-accordion-section-border-color: var(--utrecht-accordion-section-hover-border-color);
```

Comment:
```suggestion
  border-block-start-color: var(--utrecht-accordion-section-border-color, currentColor);
  border-block-start-style: solid;
  border-block-start-width: var(--utrecht-accordion-section-border-width, 0);
  break-inside: avoid;
  &:is(:last-child, :only-child) {
    border-block-end-color: var(--utrecht-accordion-section-border-color, currentColor);
    border-block-end-style: solid;
    border-block-end-width: var(--utrecht-accordion-section-border-width, 0);
  }
  &:not(:last-child) {
    border-block-end-color: transparent;
  }
  &:hover {
    --utrecht-accordion-section-border-color: var(--utrecht-accordion-section-hover-border-color, currentColor);
```

Reply:
Dit is hoe de [NL Design System componenten](https://nldesignsystem.nl/componenten/) pagina eruit ziet met deze PR zonder `0` als default:

<img width="849" alt="Screenshot 2025-05-06 at 16 48 53" src="https://github.com/user-attachments/assets/1081da0b-cbba-48e7-be1a-6886d28ecd23" />